.gitlab-ci.yml

```diff
@@ -9,17 +9,12 @@ include:
     file: github-project-pipelines.gitlab-ci.yml
 
 variables:
-  GIT_STRATEGY: none
+  CMAKE_BUILD_PARALLEL_LEVEL: 3
```

Comment:
why 3?

Reply:
This was inherited from the GitHub Actions macOS runners. It could probably be increased a bit.

Reply:
@heerener do we currently have a definite/specific number of cores allocated to each GitLab job on the M1 runner?

Reply:
As far as I can tell there is nothing explicitly configured, which means that it's up to the OS to handle this.

Reply:
We can fine-tune this later, but 3 seems reasonable if there are 8 physical cores and we allow a maximum number of concurrent jobs around 3-4 per machine.